addons/mail/static/src/components/composer/composer.scss

```diff
@@ -107,17 +107,27 @@
     justify-self: center;
 }
 
+.o_Composer_subject {
+    border-top: $border-width solid $border-color;
+    border-right: $border-width solid $border-color;
+    border-left: $border-width solid $border-color;
```

Comment:
Uh... Not your fault, but this naming of variables sticks...

Reply:
stinks* :stuck_out_tongue_winking_eye:  ?

Reply:
> stinks* 😜 ?

Woops, yeah "stinks", failed attempt at modesty... 😅